### PR TITLE
edits to discussion, conclusion, box2

### PR DESCRIPTION
**Pull request by robcrystalornelas**
In this PR, I addressed comments left in our hypothesis 'big picture' group.

There are still a couple of remaining comments that I can address once we figure out the final number of "ways to use GitHub.

- In paragraph 2, I think I lined to Figure 2 correctly?
- Added citations, will need to figure out if these all linked appropriately.
- Fixed line breaks throughout. Hopefully the paragraphs appear as paragraphs in the rendered document.
- Also added a table 1 and Box2 citations, not sure if that worked.

### CODE REVIEW

File: content/06.discussion-a.md [hunk 1]
```diff
@@ -3,61 +3,67 @@
...
+This call comes in light of both the continual shift toward open-science and increasing computational and data requirements in EEB.
+Until now, resources and practical guidance specifically focused on using GitHub within the EEB community have been dispersed in blog posts and video tutorials (Box @box:box-2).
+We felt these resources have been extremely useful for us to learn to use GitHub in our own work, and that a collation of the main ideas into one medium, while adding on our personal perspectives, would be of use in the EEB community.  
+
+In this paper, we describe 13 tractable ways that EEB researchers can leverage GitHub to enable more transparent and collaborative research (Figure @fig:scatterblob). Many of the examples are specifically meant for first-time GitHub users and can likely be adopted with just several hours of practice (storing data, creating virtual notebooks, making code citable).
+For example, storing code and data and making it citable generally just involve creating a repository on GitHub, pushing code to the repository, and then going through the necessary steps for creating a DOI for the repository.
```
Comment (DrMattG) [inline content/06.discussion-a.md:12]: is it worth adding: "necessary steps (e.g. connecting a repository with Zendodo; see below)" Or something like that here?

File: content/06.discussion-a.md [hunk 2]
```diff
@@ -3,61 +3,67 @@
...
+These actions are often covered in any introduction to GitHub tutorial and take little overhead to implement.
+On the other hand, some other examples we describe may require a greater time commitment, but have the potential to make EEB research more open, accessible, and collaborative than ever before.
+For example, managing full research projects or research labs on GitHub will require careful thought as to how to delegate tasks such as reviewing pull requests or creating issues, as well as thought as to how modular to make the research project or research lab (i.e., which repository will be used for what, and how many repositories are needed).
+Additionally, collaboratively writing a manuscript using GitHub, as we have done here, will involve a learning curve for co-authors less familiar with the intricacies of GitHub, and also require overhead to set up the repository using GitHub actions.
```
Comment (DrMattG) [inline content/06.discussion-a.md:16]: I am not sure but will check after reviewing this but I think that we changed "manuscript" to "paper" elsewhere in the document.

**Review by DrMattG — APPROVED** [review]
Looks great - I added some minor comments.

File: content/06.discussion-a.md (hunk 2 above)
Comment (DrMattG) [reply content/06.discussion-a.md:16]: Just checked and we did change manuscript to paper in the collaborative code section

**Review by BrandonEdwards — APPROVED** [review]
Looks good with Matt's changes too!

**Pull request merged**